swagger2.0/oic.wk.col.swagger.json

```diff
@@ -268,61 +283,77 @@
     "sbaseline" : {
       "properties": {
         "links" : {
-            "description": "A set of simple or individual OIC Links.",
-            "items": {
+          "description": "A set of simple or individual OIC Links.",
```

Comment:
In row 275, we have "interface-all" listed with ll, baseline and batch in that order.  Where is this called from within the schema to identify that this is the place to find the Default OCF Interface.  I thought it was supposed to be referenced in a "get" request/response somewhere.

In other words, what leads a Client/CTT to this place to find the Default OCF Interface?

Reply:
This is for DMWG to finalize.  But to repeat, we use interface-all as a marker FOR THE CTT, it is what it is currently looking for.  It isn't referenced in the schema path sections.